src/mame/pacman/pacman.cpp

```diff
@@ -8582,7 +8609,7 @@ void pacman_state::init_mspackpls()
  *
  *************************************/
 
-//          rom       parent    machine   inp       state          init
+//          ROM       PARENT    MACHINE   INPUT     STATE          INIT
```

Comment:
Please don’t do this.  excessive capitals are like screaming, and it doesn’t make it easier to read.

Reply:
Reverted!